packages/eslint-plugin/src/rules/prefer-regexp-exec.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * @fileoverview Prefer RegExp#exec() over String#match()
```

Comment:
This should match the description in https://github.com/typescript-eslint/typescript-eslint/pull/305/files#diff-c30b95a0b8fd3516e0047b92411ee36fR19